examples/seq2seq/run_seq2seq.py

```diff
@@ -336,8 +349,10 @@ def main():
     # We need to tokenize inputs and targets.
     if training_args.do_train:
         column_names = datasets["train"].column_names
-    else:
+    elif training_args.do_eval:
         column_names = datasets["validation"].column_names
+    else:
```

Comment:
Maybe do an `elif training_args.do_predict` here and have an else that prints something like "There is nothing to do, please pass along --do_train, --do_eval and/or --do_predict." and returns early?